modules/cluster-logging-collector-log-forward-about.adoc

```diff
@@ -5,27 +5,27 @@
 [id="cluster-logging-collector-log-forward-about_{context}"]
 = Understanding the Log Forwarding API
 
-Forwarding cluster logs using the Log Forwarding API requires a combination of _outputs_ and _pipelines_ to send logs to specific endpoints inside and outside of your {product-title} cluster. 
+Forwarding cluster logs using the Log Forwarding API requires a combination of _outputs_ and _pipelines_ to send logs to specific endpoints inside and outside of your {product-title} cluster.
 
 [NOTE]
 ====
-If you want to use only the default internal {product-title} Elasticsearch logstore, do not configure any outputs and pipelines. 
-==== 
+If you want to use only the default internal {product-title} Elasticsearch logstore, do not configure any outputs and pipelines.
+====
 
-An _output_ is the destination for log data and a pipeline defines simple routing for one source to one or more outputs. 
+An _output_ is the destination for log data and a pipeline defines simple routing for one source to one or more outputs.
 
 An output can be either:
 
-* `elasticsearch` to forward logs to an external Elasticsearch v5.x cluster, specified by server name or FQDN, and/or the internal {product-title} Elasticsearch logstore. 
+* `elasticsearch` to forward logs to an external Elasticsearch v5.x, Elasticsearch v6.x, or Elasticsearch v7.x cluster, specified by server name or FQDN, and/or the internal {product-title} Elasticsearch logstore.
```

Comment:
I noticed the update is for logging 4.5. In Logging 4.5, our internal ES version is 6.8. 
I tested logforwarding with logging 4.5, here are the details:
1. forward logs to ES 5.6.16:
logs can't be forwarded to ES, lots of errors in fluentd:
```
2021-05-11 03:16:39 +0000 [warn]: dump an error event: error_class=Fluent::Plugin::ElasticsearchErrorHandler::ElasticsearchError error="400 - Rejected by Elasticsearch [error type]: invalid_type_name_exception [reason]: 'Document mapping type name can't start with '_', found: [_doc]'" location=nil tag="kubernetes.var.log.containers.centos-logtest-2kpgp_dch-y_centos-logtest-d1224212e0c72468a1d4153c3c30387911209bb9761934c5f6db3f8cad196718.log" time=2021-05-11 03:15:07.396426896 +0000 record={"docker"=>{"container_id"=>"d1224212e0c72468a1d4153c3c30387911209bb9761934c5f6db3f8cad196718"}, "kubernetes"=>{"container_name"=>"centos-logtest", "namespace_name"=>"dch-y", "pod_name"=>"centos-logtest-2kpgp", "container_image"=>"quay.io/openshifttest/ocp-logtest@sha256:f23bea6f669d125f2f317e3097a0a4da48e8792746db32838725b45efa6c64a4", "container_image_id"=>"quay.io/openshifttest/ocp-logtest@sha256:f23bea6f669d125f2f317e3097a0a4da48e8792746db32838725b45efa6c64a4", "pod_id"=>"fe4123cb-27ee-466d-be65-87935d20f2fb", "host"=>"qitang-jpbfw-worker-c-q99xl.c.openshift-qe.internal", "master_url"=>"https://kubernetes.default.svc", "namespace_id"=>"79629354-c22d-4096-9034-070fb0821de0", "namespace_labels"=>{"kubernetes_io/metadata_name"=>"dch-y"}, "flat_labels"=>["run=centos-logtest", "test=centos-logtest"]}, "message"=>"{\"message\": \"MERGE_JSON_LOG=true\", \"level\": \"debug\",\"Layer1\": \"layer1 0\", \"layer2\": {\"name\":\"Layer2 1\", \"tips\":\"Decide by PRESERVE_JSON_LOG\"}, \"StringNumber\":\"10\", \"Number\": 10,\"foo.bar\":\"Dot Item\",\"{foobar}\":\"Brace Item\",\"[foobar]\":\"Bracket Item\", \"foo:bar\":\"Colon Item\",\"foo bar\":\"Space Item\" }", "level"=>"unknown", "hostname"=>"qitang-jpbfw-worker-c-q99xl.c.openshift-qe.internal", "pipeline_metadata"=>{"collector"=>{"ipaddr4"=>"10.0.128.3", "inputname"=>"fluent-plugin-systemd", "name"=>"fluentd", "received_at"=>"2021-05-11T03:15:07.833206+00:00", "version"=>"1.7.4 1.6.0"}}, "@timestamp"=>"2021-05-11T03:15:07.396426+00:00", "viaq_index_name"=>"app-write", "viaq_msg_id"=>"Y2IxYTZlNzMtOWM1Yy00NmYwLTg5MDktZGVmODBlMjJlNmEw", "openshift"=>{"labels"=>{"forward-with-labels"=>"forward-to-insecure-es"}}}
``` 
2. forward logs to ES 6.8.15/6.8.4:
works as expected
3. forward logs to ES 7.12.1:
logs can be received at the beginning, but a few minutes later, the docs.count in elasticsearch stops increasing, and fluentd keeps repeating a warning message: `warning: 299 Elasticsearch-7.12.1-3186837139b9c6b6d23c3200870651f10d3343b7 "[types removal] Specifying types in bulk requests is deprecated."`

Per the above results, I think we'd better only write ` Elasticsearch v6.x` here.